.azure/pipelines/components-e2e-tests.yml

```
@@ -44,9 +44,9 @@ jobs:
       displayName: NPM install
     - script: .dotnet/dotnet build ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-restore
       displayName: Build
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter QuarantinedTest\!~github --logger trx
+    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter 'Quarantined!=true|Quarantined=false' --logger trx
       displayName: Run E2E tests
-    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter QuarantinedTest~github -p:RunQuarantinedTests=true --logger trx
+    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter 'Quarantined=true' -p:RunQuarantinedTests=true --logger trx --logger:"trx%3BLogFileName=$(_TestResultTrxFileName)" --logger:"html%3BLogFileName=$(_TestResultHtmlFileName)" "--ResultsDirectory:$(_TestResultDirectory)"`
```

I doubt this command will work without setting `$(_TestResultTrxFileName)`, `$(_TestResultHtmlFileName)`, and `$(_TestResultDirectory)`. My suggestion wasn't meant to be complete (see comments with it).

---

Failed at least in part due to the backtick I accidentally included in my suggestion. How about
```suggestion
    - script: .dotnet/dotnet test ./src/Components/test/E2ETest -c $(BuildConfiguration) --no-build --filter 'Quarantined=true' -p:RunQuarantinedTests=true
                 --logger:"trx%3BLogFileName=Microsoft.AspNetCore.Components.E2ETests.trx"
                 --logger:"html%3BLogFileName=Microsoft.AspNetCore.Components.E2ETests.html"
                 "--ResultsDirectory:$(Build.SourcesDirectory)/artifacts/TestResults/$(_BuildConfig)/Quarantined"
```